paseos/geometric_model/geometric_model.py

```diff
@@ -0,0 +1,102 @@
+                Coordinates of the corners of the object. If not selected, it will default to a cube that can be scaled
+                by the scale. Uses Trimesh to create the mesh from this and the list of faces.
+            faces (list): List of the indexes of the vertices of a face. This builds the faces of the satellite by
+                defining the three vertices to form a triangular face. For a cuboid each face is split into two
+                triangles. Uses Trimesh to create the mesh from this and the list of vertices.
+            scale (float): Parameter to scale the cuboid by, defaults to 1.
+        """
+        logger.trace("Initializing cuboid geometrical model.")
+
+        self._actor = local_actor
+        self._actor_mass = actor_mass
+        self.vertices = vertices
+        self.faces = faces
+        self.scale = scale
+
+    def set_mesh(self):
+        """Creates the mesh of the satellite. If no vertices input is given, it defaults to a cuboid scaled by the
+        scale value. The default without scale values is a cube with 1m sides. This uses the python module Trimesh.
+
+        Returns:
+            mesh: Trimesh mesh of the satellite
+        """
+        if self.vertices is None:
+            # Defines the corners of the mesh, values are in meters, from the origin of the body frame.
+            self.vertices = [
+                [-0.5, -0.5, -0.5],
+                [-0.5, -0.5, 0.5],
+                [-0.5, 0.5, -0.5],
+                [-0.5, 0.5, 0.5],
+                [0.5, -0.5, -0.5],
+                [0.5, -0.5, 0.5],
+                [0.5, 0.5, -0.5],
+                [0.5, 0.5, 0.5],
+            ]
+            # List of three vertices to form a triangular face of the satellite.
+            # Two triangular faces are used per side of the cuboid.
+            self.faces = [
+                [0, 1, 3],
+                [0, 3, 2],
+                [0, 2, 6],
+                [0, 6, 4],
+                [1, 5, 3],
+                [3, 5, 7],
+                [2, 3, 7],
+                [2, 7, 6],
+                [4, 6, 7],
+                [4, 7, 5],
+                [0, 4, 1],
+                [1, 4, 5],
+            ]
+
+        mesh = trimesh.Trimesh(self.vertices, self.faces)
+        # Scales the mesh by the scale factor.
+        self._actor_mesh = mesh.apply_scale(self.scale)
+        return self._actor_mesh
+
+    @property
+    def find_moment_of_inertia(self):
+        """Gives the moment of inertia of the actor, assuming constant density.
+
+        Returns:
+            np.array: Mass moments of inertia for the actor
+
+        I is the moment of inertia, in the form of [[Ixx Ixy Ixz]
+                                                    [Iyx Iyy Iyx]
+                                                    [Izx Izy Izz]]
+        """
+        self._actor_moment_of_inertia = self._actor_mesh.moment_inertia
+        return self._actor_moment_of_inertia
```

Comment:
this function doesn't do anything right now, so please raise a NotImplemented error so it's clear that nothing happens